subscriber/src/subscriber/subscriber.bal

```diff
@@ -71,21 +76,36 @@ public function main (string secret,                // secret to send to the hub
     // check whether this version is still supported
     hr = check hc->get("/isactive/" + MY_VERSION);
     if hr.statusCode != 200 {
-        io:println("*** This version of the subscriber is no longer supported!");
-        return;
+        return error("*** This version of the subscriber is no longer supported!");
```

Comment:
@sanjiva, hope this change is OK. Updated to return an error so that the program terminates.

Reply:
Thanks!